### DISCUSSION TITLE
2024-06-08(토)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#117] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by minyoung0503** [comment]
### 완료한 일
공동구매 전체보기 페이지

### 해야할 일
별점 text 추가
마이페이지 -> 리뷰 상세 페이지 UI

### 어려운 일

**Comment by devwqc** [comment]
### 완료한 일
- 헤더 + 레이아웃 최종 작업
- Toast 컴포넌트(레이아웃에 따라서 토스트 위치 변경 작업 필요)

### 해야할 일
- 홈 페이지 및 상품 목록 리스트

### 어려운 일
- 레이아웃에 따라서 토스트 위치를 동적으로 옮기는 작업이 어렵네요...

**Comment by Seoin02** [comment]
### 완료한 일
- 로그인된 my 페이지 UI

### 해야할 일
- my/order 페이지 UI
- 무한스크롤 커스텀 훅 구현

### 어려운 일
- 토큰 기반 유저 관리는 정훈님과 계속 소통하면서 작업하겠습니다. 서버 연결하면 작업하기 더 수월할 것 같아요! 그래서 연결해서 작업하려고요ㅎㅎ
